server/src/routes/scheduleRoutes.js

```diff
@@ -0,0 +1,188 @@
+ *                 description: date of events for the schedule
+ *                 example: Monday September 5
+ *               events:
+ *                 type: array
+ *                 description: array of events occuring on the date of schedule
+ *                 items:
+ *                   type: object
+ *                   required:
+ *                     - name
+ *                     - description
+ *                     - time
+ *                   properties:
+ *                     name:
+ *                       type: string
+ *                       description: name of the event
+ *                       example: Matriculation
+ *                     description:
+ *                       type: string
+ *                       description: description of the event
+ *                       example: lorem ipsum
+ *                     time:
+ *                       type: string
+ *                       description: time of the event
+ *                       example: 1:00 pm - 2:00pm
+ *                 minItems: 1
+ *     responses:
+ *       '200':
+ *         description: Schedule was successfully edited
+ *         content:
+ *           application/json:
+ *             schema:
+ *                $ref: '#components/schemas/Schedule'
+ */
+router.put(
+  '/edit',
+  checkLoggedIn,
+  hasAuthScopes(['schedule:edit']),
+  ScheduleController.editSchedule,
+);
+
+/**
+ * @swagger
+ * /schedule/reorder:
+ *   put:
+ *     summary: reorder the events of a schedule object
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               id:
+ *                 type: string
+ *                 example: 62abaed97fb4951f642a8ea9
+ *               order:
+ *                 type: array
+ *                 items:
+ *                   type: number
+ *                   minimum: 0
+ *                   example: [2,1,3,4,0]
+ *     responses:
+ *       '200':
+ *         description: Schedule was successfully reordered
+ *         content:
+ *           application/json:
+ *             schema:
+ *                $ref: '#components/schemas/Schedule'
+ */
+router.put(
+  '/reorder',
+  checkLoggedIn,
+  hasAuthScopes(['schedule:edit']),
+  ScheduleController.reorderSchedule,
+);
+
+/**
+ * @swagger
+ * /schedule/delete:
+ *   delete:
+ *     summary: delete a schedule object
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               id:
+ *                 type: string
+ *                 example: 62abaed97fb4951f642a8ea9
+ *     responses:
+ *       '200':
+ *         description: Schedule was successfully deleted
+ *         content:
+ *           application/json:
+ *             schema:
+ *                $ref: '#components/schemas/Schedule'
+ */
+router.delete(
+  '/delete',
```

Comment:
```suggestion
  '/delete/:id',
```
Again, for delete requests using the request body is an anti-pattern (google cloud will automatically reject the request, as an example). So, you can use a url parameter to specify which schedule to delete. However, I don't think it makes sense to have an endpoint to delete the entire schedule - you are going to want to have one which deletes a specific event from a schedule